src/layouts/AppLayout.tsx

```diff
@@ -1,6 +1,6 @@
 import { Outlet } from "react-router-dom";
-import Footer from "./Footer";
-import Header from "./Header";
+import Footer from "../components/Footer";
+import Header from "../components/Header";
 
 function Applayout(){
     return(
```

Comment:
Applayout can be theme based , kindly check utility console rootlayout sample too , i know it depends on an api config but still you can have your rendered concept in the else